### PR TITLE
Add QR Code information to markdown

### PR DESCRIPTION
**Pull request by sebatze**
the ha-rq-code tag is undocumented. This should be added

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced QR code generation support within the Markdown card.
	- Added documentation for the new `ha-qr-code` component, including configuration parameters and implementation examples.
	- Included a visual example of the QR code within the Markdown card.

### CODE REVIEW

**Review by sebatze — COMMENTED** [review]
would be nice to add an image (unfortunately, I am not allowed to upload images to /images/dashboards/markdown_ha-alert.png

But a screenshot of the rendered markdown code would be great

File: source/_dashboards/markdown.markdown [hunk 1]
```diff
@@ -150,3 +150,33 @@ content: |
   <ha-alert alert-type="success">This is a success alert — check it out!</ha-alert>
   <ha-alert title="Test alert">This is an alert with a title</ha-alert>
 ```
+
```
Comment (c0ffeeca7) [inline source/_dashboards/markdown.markdown:153]:
```suggestion
```

File: source/_dashboards/markdown.markdown [hunk 2]
```diff
@@ -150,3 +150,33 @@ content: |
...
+
+## ha-qr-code
+
+You can also create QR-Codes in the Markdown card.
+
+
```
Comment (c0ffeeca7) [inline source/_dashboards/markdown.markdown]:
```suggestion
```

File: source/_dashboards/markdown.markdown [hunk 3]
```diff
@@ -150,3 +150,33 @@ content: |
...
+Available parameters:
+- data: The actual data to encode in the QR-Code
+- scale: A scale factor for the QR code, default is 4
+- width: Width of the QR code in pixels
+- margin: A margin around the QR code
+- error-correction-level: low; medium; quartile; high)
+- center-image: An image to place on top of the qr code (might need a higher error-correction-level)
+
```
Comment (c0ffeeca7) [inline source/_dashboards/markdown.markdown:165]:
```suggestion
```

File: source/_dashboards/markdown.markdown [hunk 4]
```diff
@@ -150,3 +150,33 @@ content: |
...
+type: markdown
+content: >-
+  <ha-qr-code data='hallo' width="180"></ha-qr-code>
+
+  <ha-qr-code data='hallo' scale="6" margin="0"
+  center-image="/static/icons/favicon-192x192.png"></ha-qr-code>
+
+  <ha-qr-code data='hallo' error-correction-level="quartile" scale="6"
+  center-image="https://brands.home-assistant.io/_/tuya/icon@2x.png"></ha-qr-code>
+
```
Comment (c0ffeeca7) [inline source/_dashboards/markdown.markdown]:
```suggestion
```

**Comment by c0ffeeca7** [comment]
> would be nice to add an image (unfortunately, I am not allowed to upload images to /images/dashboards/markdown_ha-alert.png
> 
> But a screenshot of the rendered markdown code would be great

what do you mean, you are not allowed? you should be able to just add it to the image folder and include it in the PR. You selected /source/images/dashboards/...?

**Review by c0ffeeca7 — APPROVED** [review]
Nice! Thank you, @sevorl :+1:

**Comment by c0ffeeca7** [comment]
> would be nice to add an image (unfortunately, I am not allowed to upload images to /images/dashboards/markdown_ha-alert.png
> 
> But a screenshot of the rendered markdown code would be great


> would be nice to add an image (unfortunately, I am not allowed to upload images to /images/dashboards/markdown_ha-alert.png
> 
> But a screenshot of the rendered markdown code would be great


<https://github.com/home-assistant/home-assistant.io/pull/34987>

**Pull request merged**